src/add-delete.js

```diff
@@ -5,7 +5,17 @@ function addTask(description, activities) {
 }
 
 function deleteTask(index) {
-  let activities = [2, 'teste'];
+  let activities = ['teste2'];
+  activities.splice(index, 1);
+  activities = activities.filter((Obj, index) => {
+    Obj.index = index;
+    return true;
+  });
+  return activities;
+}
+
+function clearCompleted(index) {
+  let activities = ['teste2', 'test3'];
```

Comment:
- [ ]  You should test the functions that you created in the `functions.js` file. You have created whole new functions in this `add-delete` file, you are testing these new functions. 

**You are not properly testing your app. You should use the mocks only to mock the data or environment that you don't have control over, not your application.**